test/sdk/nodejs/testSDKClient.js

```diff
@@ -66,4 +66,95 @@ const connect = async () => {
 	}
 };
 
+const testCounts = async(sdk) {
```

Comment:
This is a port of the go test in `sdk-client-test.go`, however it is currently not executed as not sure yet how to do the equivalent of
```
if runtime.FeatureEnabled(runtime.FeatureCountsAndLists) {
```
If there is an environment variable set during the test then that could be used